pkg/netconf/tap.go

```diff
@@ -1,96 +0,0 @@
-// Copyright (c) 2022 Intel Corporation.  All Rights Reserved.
```

Comment:
This file needs to be put back for DPDK

Reply:
File restored back for DPDK target and commented out the resulting duplicate function - setLinkAddressFunc()